.github/workflows/pull-request-checks.yml

```diff
@@ -346,7 +346,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: crate-ci/typos@master
+    - uses: crate-ci/typos@v1.12.12
```

Comment:
What does `v1.12.12` mean here?

Reply:
Read this https://github.com/crate-ci/typos/issues/610. it's the release tag.